Fix view toolbar button to properly show menu when clicked

AdityaShome opened the pull request:
## Issue
The view menu from `toolbars.py` was not visible because page setup was missing.

## Fixes
- Corrected the View button setup to properly link to the `view_toolbar`.
- View button is visible now with its previous contents "zoom in" and zoom out".
- It links the view button to open or control the `view_toolbar ` and opens the panel when clicked.

![Screenshot from 2025-04-29 07-48-39](https://github.com/user-attachments/assets/067e6716-32b2-4670-962e-0703cfec4793)

chimosky commented:
Good catch! Thank you!

Could you edit your commit message to include some of the details in your opening commit.

Would also be nice to have an extra button that'll return the zoom to the normal width if it was zoomed in or out at any point, the write activity has something similar, I can't think of another activity that does at the moment, one keeps eluding me.

quozl commented:
Some of my comments in https://github.com/sugarlabs/Pippy/pull/109 also apply here.

AdityaShome commented:
Along with fixing the view menu button, I included a reset button that reverts any change in font to it's normal size.
A proper tooltip was added for it. 
Updated the commit message to include detailed changed.
Added keyboard shortcuts (ctrl+=, ctrl+- and ctrl+0).
I moved the dark mode and zoom buttons in view menubar as said in #109.

![Screenshot from 2025-04-30 08-21-41](https://github.com/user-attachments/assets/26509e08-eaec-4526-a8ba-dd23385a61e5)

chimosky commented:
What activity did you see using that icon for that purpose?

AdityaShome commented:
Thanks for the detailed feedback!
- I've updated the commit message to follow the one-line summary format as per the [contributing guidelines](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits).
- Moved zoom out before zoom in button to match the format of browse activity.
- Changed tooltip "Reset zoom" to "Actual size".
- Set the accelarator to ctrl++ and also responding to ctrl+=.
- ctrl+0 is working for both editor and terminal.

chimosky commented on toolbars.py:
```diff
@@ -62,3 +84,33 @@ def _font_size_increase(self, button):
+        ctrl = event.state & Gdk.ModifierType.CONTROL_MASK
+        
+        if ctrl:
+            # Try multiple approaches to catch the plus key
+            if (event.keyval == ord('+') or 
+                event.keyval == Gdk.KEY_plus or
+                event.keyval == Gdk.KEY_KP_Add or
+                event.keyval == Gdk.KEY_equal):
```
I don't think we need multiple approaches for this.

AdityaShome replied:
Sure, I kept one approach responding to ctrl++, also ctrl+=(on most keboards).

chimosky commented on toolbars.py:
```diff
@@ -62,3 +84,33 @@ def _font_size_increase(self, button):
+                event.keyval == Gdk.KEY_KP_Add or
+                event.keyval == Gdk.KEY_equal):
+                
+                # Print debug info to help diagnose the issue
+                print("Caught key press: ", event.keyval)
```
Unnecessary comment and print statement.

The activity uses logging that you should take advantage of if you need to, we don't need to in this case.

AdityaShome replied:
Last commit removes the comments and print statement as it's logging is not currently required. I'll use the activity's logging system if needed in the future.
